sdk/java/src/main/java/net/consensys/gpact/messaging/common/attestorrelayer/AttestorRelayerWebApi.java

```diff
@@ -94,7 +98,7 @@ public static void setupObserver(
   public static void stopObserver(String observerUrl) throws CrosschainProtocolStackException {
     LOG.info("StopObserver: ObserverURL: {}", observerUrl);
 
-    Bytes type = Bytes.of(STOP_OBSERVE_REQ_TYPE);
+    Bytes type = Bytes.of(STOP_OBSERVER_REQ_TYPE);
```

Comment:
Because this is meant to stop the multi-source observer entirely and not a specific observer that it manages, the type is correct. The 'START_OBSERVATION_REQ_TYPE` used in the `setupObserver()` function however is instructing the multi-source observer to create and start a specific observer.